docs/guides/references/_changelog-list.mdx

```diff
@@ -0,0 +1,284 @@
+## 10.7.0
```

Comment:
Is this file auto-generated or will updating this need to be a part of the release process?

Reply:
It's not autogenerated atm. That's a cool thing to add but it's sadly a part of the hack to get this TOC on the page.